project/externals/cachelib.cmake

```diff
@@ -9,6 +9,7 @@ ExternalProject_Add(
     ${name}
     GIT_REPOSITORY https://github.com/facebook/CacheLib.git
     GIT_TAG 12822a2eddd04bb3e252f458b36ee0675a2d4643  # As of 2021/12/02
+    GIT_SUBMODULES ""
```

Comment:
why add this line?

Reply:
This empty string combined with policy CMP0097 will avoid pulling all sub-modules. We don't use the libraries from the sub-modules anyway